server/controllers/auth.js

```diff
@@ -92,6 +93,8 @@ exports.loginUser = asyncHandler(async (req, res, next) => {
           name: user.name,
           email: user.email,
         },
+        profile,
```

Comment:
Ya this will do it! lets see what ethan will say!

Reply:
This is great! Thanks!